### PR TITLE
refinement towards sync_cp_lag_too_behind_policy to avoid submit sync…

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #25441

### CODE REVIEW

**Comment by MrPresent-Han** [comment]
rerun ut

File: internal/util/paramtable/component_param.go [hunk 1]
```diff
@@ -1811,6 +1813,10 @@ func (p *dataNodeConfig) initCpLagPeriod() {
 	p.CpLagPeriod = time.Duration(cpLagPeriod) * time.Second
 }
 
+func (p *dataNodeConfig) initCpSyncLimit() {
+	p.CpLagSyncLimit = p.Base.ParseUInt16WithDefault("datanode.segment.cpLagSyncLimit", 10)
```
Comment (xiaofan-luan) [inline internal/util/paramtable/component_param.go:1817]: try not to use u16 as config because this may overflow under extreme cases

**Pull request merged**